index.html

```diff
@@ -143,7 +143,50 @@ <h2 id="projects-h2">My Recent <br id="break" />Works</h2>
           <hr id="rule" />
         </div>
         <div id="projects-container">
-          <!-- project one -->
+          <!-- project one
+          <div class="popup-wrapper">
+            <div class="popup">
+              <div class="popup-content">
+                  <p class="close-popup"><i class="fa fa-times" aria-hidden="true"></i></p>
+                <div class="popup-image">
+                </div>
+                <div class="popup-text">
+                  <h2 class="popup-header">Keeping track of <br id="br1"> hundreds of <br>
+                    components</h2>
+                    <ul class="technologies">
+                      <li>Ruby on rails</li>
+                      <li>css</li>
+                      <li>JavaScript</li>
+                    </ul>
+                    <ul class="technologies-desktop">
+                      <li>Codekit</li>
+                      <li>Github</li>
+                      <li>JavaScript</li>
+                      <li>Bootstrap</li>
+                      <li>Terminal</li>
+                      <li>Codepen</li>
+
+                    </ul>
+                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+                      when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                      Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the 
+                      release</p>
+                      <div class="popup-buttons">
+                        <button type="button"> 
+                          <a href="https://ceemos96.github.io/Portfolio-setup-and-mobile-version-skeleton/" target="_blank"> 
+                            See Live <i class="fa fa-link" ></i>
+                          </a>
+                        </button>
+                        <button type="button">
+                          <a href="https://github.com/Ceemos96/Portfolio-setup-and-mobile-version-skeleton" target="_blank"> 
+                            See Source  <i class="fab fa-github"></i>
+                          </a>
+                        </button>
+                      </div>
+                </div>
+              </div>
+            </div>
+          </div>
```

Comment:
**[REQUIRED CHANGES]**

- [ ] Please, kindly remove this unused commented code all through to where it ends in your `html file`, it makes your codebase not clean and makes you not a professional.